src/cmd/factory/impl/common.go

```diff
@@ -250,7 +250,7 @@ func generateAndRestoreDriveItems(
 		d, err := ctrl.AC.Stable.
 			Client().
 			Sites().
-			BySiteIdString(protectedResource.ID()).
```

Comment:
Make up your mind, Microsoft!

(honestly though, i'm glad they rolled this back.  Was a silly idea)